### ISSUE TITLE
[1.12.2] Soymilk and Presser

### ISSUE THREAD

**Issue by TimGoll** [MatrexsVigil/harvestcraft#80]
If I autoinsert soybeans into the presser, the output slots get filled with soybeans, too
and there is no JEI support for your custom machines

1.12.2, latest of everything

EDIT: JEI fixed with this: https://minecraft.curseforge.com/projects/just-enough-harvestcraft

**Comment by TimGoll** [comment]
There are also no item tooltips in your machines.
Add this to your clientGui Class
```java
@Override
public void drawScreen(int mouseX, int mouseY, float partialTicks) {
    super.drawScreen(mouseX, mouseY, partialTicks);
    renderHoveredToolTip(mouseX, mouseY);
}
```

**Comment by tabbykitty1** [comment]
It may depend on which version of Forge you are using or version of HarvestCraft. I do not have this problem and 1.12.2 everything also, and do not use "JEHC". Soybean in Presser gives me Tofu and SoyMilk.

Minecraft 1.12.2
HarvestCraft 1.12.2a
Forge 1.12.2-14.23.0.2491

**Comment by TimGoll** [comment]
It works, yes
but hoppers (for example) insert into all three slots, if they are all empty

**Comment by TimGoll** [comment]
JEHC fixes only the recipes in JEI, without this mod, the custom crafting methods aren't shown in JEI

**Comment by ghost** [comment]
You mean with hoppers? It is currently messed up in hoppers.

**Comment by TimGoll** [comment]
Hoppers and pipes, yes

**Comment by Shindur** [comment]
Can confirm, still happening with mekanism pipes as well, even using a logistical sorter and setting it to only output 64, but fills all 3 slots.

**Comment by TimGoll** [comment]
@MatrexsVigil 
if you accept PR, I'll fix this and a few other small bugs in the next few days

**Comment by MatrexsVigil** [comment]
@TimGoll Yes, I accept PR requests.

**Comment by MatrexsVigil** [comment]
Fixed in 1.12.2g
